metadata.json

```diff
@@ -34,7 +34,7 @@
     },
     {
       "name": "puppet/python",
-      "version_requirement": ">= 1.10.0 < 5.0.0"
+      "version_requirement": ">= 5.0.0"
```

Comment:
This version number needs to be upper bounded to next major version, otherwise we lose our approved status.